### PR TITLE
Matlab: remove non-ascii characters (Matlab 2020a)

### PR DESCRIPTION
**Pull request by afraz-ahmed**
Please also see https://github.com/bear-rsg/easybuild-easyconfigs/pull/397.

### CODE REVIEW

File: easybuild/easyblocks/m/matlab.py [hunk 1]
```diff
@@ -87,6 +87,11 @@ def configure_step(self):
             write_file(licfile, lictxt)
 
         try:
+            # remove non-ascii characters from installer_input.txt
```
Comment (edmondac) [inline easybuild/easyblocks/m/matlab.py:90]: I think you should move the preconfigopts command to this easyblock, and remove it from the easyconfig.

I.e. this section becomes:
```
if LooseVersion(self.version) >= LooseVersion('2020a'):
    # remove non-ascii characters from installer_input.txt
    cmd = 'LANG=C sed -i "s/[\d128-\d255]//g" installer_input.txt'
    run_cmd(cmd, log_all=True, simple=False)
```

**Review by edmondac — CHANGES_REQUESTED** [review]
See https://github.com/bear-rsg/easybuild-easyblocks/pull/49/files#r524420263

File: easybuild/easyblocks/m/matlab.py (hunk 1 above)
Comment (orbsmiv) [reply easybuild/easyblocks/m/matlab.py:90]: @edmondac / @afraz-ahmed is this an issue specific to 2020a and above or does it occur due to running Easybuild with Python 3?

Comment (afraz-ahmed) [reply easybuild/easyblocks/m/matlab.py:90]: It's a Python 3 issue, and the non-ascii character seems to be in previous Matlab versions as well, so I'll get rid of the if statement.

File: easybuild/easyblocks/m/matlab.py [hunk 2]
```diff
@@ -87,6 +87,10 @@ def configure_step(self):
...
+            cmd = 'LANG=C sed -i "s/[\d128-\d255]//g" installer_input.txt'
```
[the remark this answers is not in the document]
Comment (afraz-ahmed) [reply easybuild/easyblocks/m/matlab.py:91]: this works... (unix command, not python)

**Review by edmondac — APPROVED** [review]
lgtm

File: easybuild/easyblocks/m/matlab.py (hunk 1 above)
Comment (edmondac) [reply easybuild/easyblocks/m/matlab.py:90]: ok

**Pull request merged**